src/main/java/com/verily/genomewarp/utils/GenomeWarpUtils.java

```diff
@@ -45,6 +55,17 @@
   // Given ~4M variants in a whole human genome (~3B base pairs) implies that each bucket will have
   // BUCKET_SORT_SIZE / (3e9 / 4e6) = 133 variants in it.
   private static final long BUCKET_SORT_SIZE = 100_000L;
+  public static final Pattern dnaSequence = Pattern.compile("[ACTGactg]+");
+  public static final int VARIANT_CONTEXT_SIZE = 5;
+
+  public static void fail(String message) {
```

Comment:
This circular dependency to access the GenomeWarpSerial logger is undesirable. Better to just have each class with its own logger (provides more hints to someone debugging anyway)